src/Conference.tsx

```diff
@@ -344,15 +351,16 @@ const Conference: React.FC = (props) => {
             },
             options
           );
+
           try {
+            const mediaDevice = await getAudioAndVideoDevice();
+            console.log("Media Devices: " + JSON.stringify(mediaDevice))
```

Comment:
Not sure if you want to leave this in or not...

Reply:
I might leave this one in just because it'll be nice for debugging what devices are being pulled by default